python/utils/__init__.py

```diff
@@ -1,5 +1,9 @@
 from utils import colorText, str2bool, find_moose_executable, runExe, check_configuration
-from MooseYaml import MooseYaml
+
+try:
+    from MooseYaml import MooseYaml
+except:
+    pass
```

Comment:
You get an error from python, cannot find library yaml error, which is usually enough for me.